graph/tags.go

```diff
@@ -222,6 +223,15 @@ func (store *TagStore) Set(repoName, tag, imageName string, force bool) error {
 	var repo Repository
 	repoName = registry.NormalizeLocalName(repoName)
 	if r, exists := store.Repositories[repoName]; exists {
+
+		if id, exists := store.Repositories[repoName][tag]; exists {
+
+			if id != img.ID && job != nil && job.Name == "load" {
+				str := "The image " + repoName + ":" + tag + " already exists, renaming the old one with id " + id[:12] + " to <none>:<none>\n"
+				job.Stdout.Write([]byte(str))
+			}
+		}
+
 		repo = r
 		if old, exists := store.Repositories[repoName][tag]; exists && !force {
```

Comment:
Looks like there's already a check here, you should be able to re-use this block of code.

also note that it's not necessary here to use `store.Repositories[repoName]` as this variable was already declared/set in the outer `if` statement as `r`.

and let's not special case the message just for the "load" job. It's useful to have in general. We can also get rid of the `job` argument to this method and just use an `io.Writer`.